code/game/objects/items/circuitboards/computer_circuitboards.dm

```diff
@@ -203,6 +203,19 @@
 	else
 		return ..()
 
+/obj/item/circuitboard/computer/cargo/express
+	name = "Express Supply Console (Computer Board)"
+	build_path = /obj/machinery/computer/cargo/express
+
+/obj/item/circuitboard/computer/cargo/express/attackby(obj/item/I, mob/user, params)//no fun allowed
+	if(istype(I, /obj/item/device/multitool))
+		to_chat(user, "<span class='notice'>NT Drop Pod saftey protocols prevent altering of the reciever spectrum.</span>")
+
+	else if(istype(I, /obj/item/card/emag))
+		to_chat(user, "<span class='notice'>NT Drop Pod saftey protocols prevent altering of routing procedures</span>")
```

Comment:
make this let it target random tiles for memes

Reply:
done wew